src/components/Icomoon.vue

```diff
@@ -57,50 +62,50 @@ export default {
     ...svgProps
   }) {
     const initialStyle = {
-      display: "inline-block",
-      stroke: "currentColor",
-      fill: "currentColor",
-    };
+      display: 'inline-block',
+      stroke: 'currentColor',
+      fill: 'currentColor',
+    }
 
-    const iconName = icon || name;
+    const iconName = icon || name
 
     const currentIcon = iconSet.icons.find(
       (item) => item.properties.name === iconName
-    );
+    )
+
+    if (!currentIcon) return {}
 
-    if (!currentIcon) return {};
+    const { width = '1024' } = currentIcon.icon
```

Comment:
It should not be single quotes.

It should also always end with a semicolon.

Reply:
Sorry, my vscode setting has overwritten the project. But I fixed the commit.